problems/strings.js

```diff
@@ -103,15 +191,70 @@ function disemvowel() {}
  * @returns {string}
  */
 
- function disemvowelLongWords(){}
+// stringYouWishToSplitIntAnArray.slice(characterYouWishToSplit)
+// Popular Splits -
+// str.split(“”) - separates every character .
+// str.split(“ “) - splits words
+
+const disemvowelLongWords = (string) => {
+  let newSentence = "";
+  let noVowels = "";
+  let newString = string.split(" ");
+  for (let i = 0; i < newString.length - 1; i++) {
+    word = newString[i]; // [cat bat lap] new string is the word
+    if (word.length >= 5) {
+      //remove vowels
+      for (let i = 0; i < word.length - 1; i++) {
+        //cat
+        letter = word[i]; //the letters in word
+        if (
+          letter === "a" ||
+          letter === "e" ||
+          letter === "i" ||
+          letter === "o" ||
+          letter === "u"
+        ) {
+          noVowels += "";
+        } else {
+          noVowels += letter;
+        }
+      }
+      
+    }
+  
+    if(newString.length - 1 === i){
+      newSentence += noVowels
+    }else{
+      newSentence += noVowels + " ";
+    }
```

Comment:
You might want to keep just line 228 here and if we reach the end of the function ( after the for-loop)  concatenate the last `noVowels` variable to the `newSentence`  variable